src/ppx/grfx/vk/vk_render_pass.cpp

```diff
@@ -133,6 +134,16 @@ Result RenderPass::CreateRenderPass(const grfx::internal::RenderPassCreateInfo*
     vkci.dependencyCount        = 1;
     vkci.pDependencies          = &subpassDependencies;
 
+    VkRenderPassMultiviewCreateInfo multiviewInfo = {VK_STRUCTURE_TYPE_RENDER_PASS_MULTIVIEW_CREATE_INFO};
+    if (hasMultiView && pCreateInfo->multiViewState.viewMask > 0) {
```

Comment:
I think we should assert that hasMultiview is true if multiviewState was provided. Otherwise it will silently ignore multiview state?